kommonitor-importer-core/src/main/java/org/n52/kommonitor/importer/io/datasource/APIFeaturesRetriever.java

```diff
@@ -0,0 +1,255 @@
+
+        if (proxyHost != null && proxyPort != null) {
+            httpHelper = HttpHelper.getProxyHttpHelper(proxyHost, proxyPort);
+        } else {
+            httpHelper = HttpHelper.getBasicHttpHelper();
+        }
+
+    }
+
+    @Override
+    protected String initType() {
+        return TYPE;
+    }
+
+    @Override
+    protected Set<DataSourceParameter> initSupportedParameters() {
+        Set<DataSourceParameter> parameters = new HashSet<>();
+        parameters.add(new DataSourceParameter(PARAM_BBOX, PARAM_BBOX_DESC, DataSourceParameter.ParameterTypeValues.STRING));
+        parameters.add(new DataSourceParameter(PARAM_BBOX_TYPE, PARAM_BBOX_TYPE_DESC, DataSourceParameter.ParameterTypeValues.STRING));
+        parameters.add(new DataSourceParameter(PARAM_CUSTOM_FILTER, PARAM_CUSTOM_FILTER_DESC, DataSourceParameter.ParameterTypeValues.STRING, false));
+        parameters.add(new DataSourceParameter(PARAM_URL, PARAM_URL_DESC, DataSourceParameter.ParameterTypeValues.STRING));
+        return parameters;
+    }
+
+    @Override
+    public Dataset<InputStream> retrieveDataset(DataSourceDefinitionType datasource) throws DataSourceRetrieverException, ImportParameterException {
+        //
+        String url = this.getParameterValue(PARAM_URL, datasource.getParameters())
+                .map(el -> (el.endsWith("/")) ? el.substring(0, el.length() - 1) : el)
+                .map(el -> (el.endsWith("/items")) ? el : el + "/items")
+                .orElseThrow(() -> new ImportParameterException("Missing parameter: " + PARAM_URL));
+
+        String bboxType = this.getParameterValue(PARAM_BBOX_TYPE, datasource.getParameters())
+                .orElseThrow(() -> new ImportParameterException("Missing parameter: " + PARAM_BBOX_TYPE));
+
+        String bboxValue = this.getParameterValue(PARAM_BBOX, datasource.getParameters())
+                .orElseThrow(() -> new ImportParameterException("Missing parameter: " + PARAM_BBOX));
+
+        String filter = this.getParameterValue(PARAM_CUSTOM_FILTER, datasource.getParameters())
+                .orElse(null);
+
+
+        HttpGet request = null;
+        try {
+            ArrayList<FeatureCollection.Feature> features = new ArrayList<>();
+
+            String bbox;
+            Geometry referenceGeometry;
+
+            // Get spatialUnit by reference
+            if (bboxType.equals("ref")) {
+
+                // Get spatialUnit from data-management
+                String response = apiClient.getAllSpatialUnitFeaturesById(bboxValue, null, "original");
+
+                // Extract bbox for first stage filtering (intersection with bbox)
+                DefaultFeatureCollection featureCollection = (DefaultFeatureCollection) featureJSON.readFeatureCollection(response);
+                ReferencedEnvelope envelope = featureCollection.getBounds();
+                bbox = String.format("%f,%f,%f,%f", envelope.getMinX(), envelope.getMinY(), envelope.getMaxX(), envelope.getMaxY());
```

Comment:
This string formatting results in a format such as `6,067986,51,166833,6,625991,51,425020` with comma as decimal separator. However, OGC API - Feature services expects a decimal point. So a generated requests, such as https://ogc-api.nrw.de/brgnrw/v1/collections/Bohrungen/items?f=json&bbox=6%2C067986%2C51%2C166833%2C6%2C625991%2C51%2C425020 results in an error response:
```json
{
"status": 400,
"title": "Bad Request",
"detail": "The following query parameters are rejected: bbox. Valid parameters for this request are: access_token, f"
}
```

Reply:
fixed in https://github.com/KomMonitor/kommonitor-importer/pull/25/commits/19cfbec0b74ed9fac7d64b0f0b442ea0e366819a